src/backend/mi2/mi2.ts

```diff
@@ -116,19 +117,20 @@ export class MI2 extends EventEmitter implements IBackend {
 					resolve(undefined);
 				}, reject);
 			} else {
-				if (separateConsole !== undefined) {
+				if (separateConsole == "external") {
 					linuxTerm.spawnTerminalEmulator(separateConsole).then(tty => {
 						promises.push(this.sendCommand("inferior-tty-set " + tty));
 						promises.push(...autorun.map(value => { return this.sendUserInput(value); }));
 						Promise.all(promises).then(() => {
-							this.emit("debug-ready");
+							if(this.application.includes('gdb')){
+								this.emit("debug-ready");
+							}
```

Comment:
So no debug-ready for non GDB scenarios and none for win-32 any more?!?

Reply:
For now, that's the situation. In the future, we can try to add more, but it might involve more work.